packages/react-composites/src/composites/common/ControlBar/CommonCallControlBar.tsx

```diff
@@ -173,6 +177,17 @@ export const CommonCallControlBar = (props: CommonCallControlBarProps & Containe
   const onDismissCaptionsSettings = useCallback((): void => {
     setShowCaptionsSettingsModal(false);
   }, []);
+
+  /* @conditional-compile-remove(teams-meeting-conference) */
+  const openTeamsMeetingConferenceModal = useCallback((): void => {
```

Comment:
Could we do this as a toggle instead of two different handlers?

Reply:
Changed